src/renderer/components/ft-shaka-video-player/ft-shaka-video-player.js

```diff
@@ -123,13 +123,18 @@ export default defineComponent({
       type: String,
       default: null
     },
+    currentPlaybackRate: {
+      type: Number,
+      default: () => 1.0
```

Comment:
As JavaScript doesn't differentiate between integers and floating point numbers, 1.0 = 1, so we might as well write it like that (saves terser having to do it for us at build time).
```suggestion
      default: () => 1
```